src/main/java/net/imagej/ops/threshold/ThresholdNamespace.java

```diff
@@ -122,6 +126,24 @@
 		return result;
 	}
 
+	// -- binary mask --
+
+	@OpMethod(op = net.imagej.ops.threshold.binaryMask.DefaultBinaryMask.class)
```

Comment:
@dietzc Note that @LeonYang5114 and I removed the `Condition` interface and related ops. I suppose we could reintroduce it, if you see a use case for it here.